wpilibj/src/athena/java/edu/wpi/first/wpilibj/Joystick.java

```diff
@@ -178,33 +159,33 @@ protected Joystick(int port, int numAxisTypes, int numButtonTypes) {
   }
 
   /**
-   * Get the X value of the joystick. This depends on the mapping of the joystick connected to the
+   * Get the kX value of the joystick. This depends on the mapping of the joystick connected to the
```

Comment:
Changing this wording seems a bit odd to me. Furthermore, this makes it inconsistent with wpilibc.